c2corg_ui/templates/waypoint/edit.html

```diff
@@ -40,19 +40,14 @@ <h1 ng-init="id = ${waypoint_id if waypoint_id else 0}" class="text-center">
 
     ## PROGRESS BAR
     <app-progress-bar>
-      ## It won't work with ng-init
-      <span class="ng-hide">
-        {{progressBarCtrl.updateMaxSteps(waypoint.waypoint_type)}}
-        {{progressBarCtrl.waypointType = waypoint.waypoint_type}}
-      </span>
 
       <ul class="progress-bar-edit">
-        <li class="nav-step-1"ng-click="progressBarCtrl.step(1, 'waypoint',  'backwards')" ng-class="{'nav-step-selected': progressBarCtrl.currentStep == 1 }"><span class="nav-text"><span translate>waypoint</span></span><span class="bullet-container"><span class="bullet"></span></span></li>
+        <li class="nav-step-1"ng-click="progressBarCtrl.step(1, 'waypoint',  'backwards')"><span class="nav-text"><span translate>waypoint</span></span><span class="bullet-container"><span class="bullet"></span></span></li>
```

Comment:
[pedantic mode on]

    <li class="nav-step-1"ng-click="progressBarCtrl.step(1, 'waypoint',  'backwards')">

should be instead

    <li class="nav-step-1" ng-click="progressBarCtrl.step(1, 'waypoint',  'backwards')">